aws/resource_aws_guardduty_member.go

```diff
@@ -93,11 +173,21 @@ func resourceAwsGuardDutyMemberRead(d *schema.ResourceData, meta interface{}) er
 		d.SetId("")
 		return nil
 	}
+
 	member := gmo.Members[0]
 	d.Set("account_id", member.AccountId)
 	d.Set("detector_id", detectorID)
 	d.Set("email", member.Email)
 
+	status := aws.StringValue(member.RelationshipStatus)
+	d.Set("relationship_status", status)
+
+	//https://docs.aws.amazon.com/guardduty/latest/ug/list-members.html
+	d.Set("invited", false)
```

Comment:
This should be `"invite"` to match the argument being passed in so Terraform can detect whether or not it should (re-)send an invite. I'll fix this on merge.